whatlies/language.py

```diff
@@ -28,16 +28,20 @@ def _input_str_legal(string):
 
     def __getitem__(self, string):
         self._input_str_legal(string)
-        doc = self.nlp(string)
-        vec = doc.vector
+        bert = False
         start, end = 0, -1
         split_string = string.split(" ")
         for idx, word in enumerate(split_string):
             if word[0] == "[":
+                bert = True
                 start = idx
             if word[-1] == "]":
                 end = idx + 1
-        if start != 0:
-            if end != -1:
-                vec = doc[start:end].vector
-        return Embedding(string, vec)
+        if end == -1:
+            end = start + 1
+        if bert:
+            new_str = string.replace("[", "").replace("]", "")
+            return Embedding(string, self.nlp(new_str)[start].vector)
+        else:
```

Comment:
technically you don't need the else here.

Reply:
wow ok super weird, I made these local changes to fix the bugs from the issue #6. I swore I overwrote those with the fixes you've since made. But I guess something went from in my git history. 
I'll change change it back and push the changes tonight. 
(same goes for the other questions related to language.py)

Reply:
i figured it might be something like that.